.github/workflows/daily.yml

```diff
@@ -731,20 +774,20 @@ jobs:
         echo "skiptests: ${{github.event.inputs.skiptests}}"
         echo "test_args: ${{github.event.inputs.test_args}}"
         echo "cluster_test_args: ${{github.event.inputs.cluster_test_args}}"
-    # On centos7 actions/checkout@v4 does not work, so we use v3
-    # ref. https://github.com/actions/checkout/issues/1487
```

Comment:
:D Yay

Reply:
Indeed :)

What about moving to more general tagging on the checkout action instead of using a straight commit hash?

Reply:
I haven't read the issue, but apparently it was included in the openSSF score card.